DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+<figure>
+	<img src="Images/Timing/StaticMpdMustBeCovered.png" />
+	<figcaption>In a static MPD, the entire period must be covered with [=media segments=].</figcaption>
+</figure>
+
+In a dynamic MPD, a representation SHALL contain enough segment references to cover the time span of the [=period=] that intersects with the [=time shift window=].
+
+Note: [=Media segments=] only become [=available=] when their end point is within the [=availability window=]. It is a valid situation that a [=media segment=] is required to be referenced but is not yet [=available=].
+
+<figure>
+	<img src="Images/Timing/MandatorySegmentReferencesInDynamicMpd.png" />
+	<figcaption>In a dynamic MPD, the [=time shift window=] determines the set of required segment references in each representation. [=Media segments=] filled with gray need not be referenced due to falling outside the [=time shift window=], despite falling within the bounds of a [=period=].</figcaption>
+</figure>
+
+Advisement: A dynamic MPD must remain valid for its entire validity duration after publishing. In other words, a dynamic MPD SHALL supply enough segment references to allow the [=time shift window=] to extend to `now + MPD@minimumUpdatePeriod`, where `now` is the current time according to [[#timing-sync|the synchronized clock]].
+
+An unnecessary segment reference is one that is not defined as required by this chapter.
+
+In a static MPD, a representation SHALL NOT contain unnecessary segment references, except when using [=indexed addressing=] in which case such segment references MAY be present.
+
+In a dynamic MPD, a representation SHALL NOT contain unnecessary segment references except when any of the following applies, in which case an unnecessary segment reference MAY be present:
+
+1. The segment reference is for future content and will eventually become necessary.
+1. The segment reference is defined via [=indexed addressing=].
+1. The segment reference is defined by an `<S>` element that defines multiple references using `S@r`, some of which are necessary.
+1. Removal of the segment reference is not allowed by [[#timing-mpd-updates-remove-content|content removal constraints]].
+
+There SHALL NOT be gaps or overlapping [=media segments=] in a representation.
+
+Clients SHALL NOT present any samples from [=media segments=] that are entirely outside the [=period=], even if such [=media segments=] are referenced.
+
+<figure>
+	<img src="Images/Timing/SamplesOnPeriodBoundary.png" />
+	<figcaption>[=Media segments=] and samples need not align with [=period=] boundaries. Some samples may be entirely outside a [=period=] (marked gray) and some may overlap the [=period=] boundary (yellow).</figcaption>
+</figure>
+
+[=Media segment=] start/end points MAY be unaligned with [=period=] start/end points except when using [=simple addressing=], which requires the first [=media segment=] start point to match the [=period=] start point.
```

Comment:
From @haudiobe: This is not correct. The presentationTimeOffset even allows to be unaligned

Reply:
I request clarification on what is not correct. Everything in this paragraph looks sound to me.

The simple addressing limitation on alignment will be removed in a future version of MPEG-DASH, as I understand it. Is that what you refer to? I think we cannot integrate that until it is actually published by MPEG, though.